views/create-capability-success.html

```diff
@@ -0,0 +1,13 @@
+{% extends "templates/master.html" %}
+
+{% block pageTitle %}
+    Create Capability
+{% endblock %}
+
+{% block content %}
+    
+<h5>Capability "{{ capabilityName }}" created successfully</h5>
+
+<a href="/" class=" bg-kainos-blue btn btn-primary text-white">Home</a>
+
+{% endblock %}
```

Comment:
if we dont have links at top of page user wont be able to go to any pages after this confirmation page